modules/ROOT/pages/plugins/premium/tinymcespellchecker.adoc

```diff
@@ -79,8 +79,6 @@ include::partial$configuration/spellchecker_select_languages.adoc[]
 
 // Bookmark for deprecated option name
 
-+++<a class="anchor" id="spellchecker_whitelist">++++++</a>+++
```

Comment:
This shouldn't have just been removed like this as we want this link to work as per the comment above. For example:
```
// Bookmark for deprecated spellchecker_whitelist option name
[[spellchecker_ignore_list]]
=== [[spellchecker_whitelist]] `spellchecker_ignore_list`
```
See: https://docs.asciidoctor.org/asciidoc/latest/sections/custom-ids/#assign-auxiliary-ids

- [ ] Add back this anchor in the `spellchecker_ignore_list.adoc` file and move the comment there as well

Reply:
Done